pkgs/top-level/all-packages.nix

```diff
@@ -19690,7 +19690,7 @@ with pkgs;
 
   libdwarf = callPackage ../development/libraries/libdwarf { };
   dwarfdump = libdwarf.bin;
-  libdwarf_0_4 = callPackage ../development/libraries/libdwarf/0.4.nix { };
+  libdwarf_20210528 = callPackage ../development/libraries/libdwarf/20210528.nix { };
```

Comment:
The filename should not contain the exact version otherwise we need to update it with every update. Using something like unstable.nix prevents this.

Reply:
Duly noted for next time, in this case it is unlikely to be an issue. This is a legacy version (so it will not be updated) that we only need for a (broken) package.

Reply:
If the package is only required for one very specific package we should think about moving it into that package to make that relation clear. This way we also don't expose a top-level attribute and definitely clean it up when it is no longer used.

Reply:
Shall we keep the package until libdwarf 2.0 released?

HHVM is using 20210528, because libdwarf 0.x has an include path different from 201xxxxx versions and the include path is expected to change back in 2.0



See https://github.com/davea42/libdwarf-code/issues/51

Reply:
I'm guessing we can keep it then ¯\_(ツ)_/¯

I have no strong opinion on this, I opened this PR after the comment made on a previous PR https://github.com/NixOS/nixpkgs/pull/194909#discussion_r990482031